extensions/binding/org.eclipse.smarthome.binding.hue/ESH-INF/thing/channels.xml

```diff
@@ -31,6 +31,81 @@
 		</tags>
 	</channel-type>
 
+	<channel-type id="last_updated">
+		<item-type>DateTime</item-type>
+		<label>Last Updated</label>
+		<description> The date when the sensor was last updated
+        </description>
+		<tags>
+			<tag>Lighting</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="lightlevel">
+		<item-type>Number</item-type>
```

Comment:
If I understand the QuantityType of Openhab like it's defined, maybe it is possible to use an implementation of the QuantityType to make the conversion between different units possible easily. Then there could be Lux and the hue-own unit from one sensor value.

Reply:
For me it is okay to have two different channels to represent the values. One for the `lightlevel` which is returned by the API and one for the `illumination`.

Maybe we can ask some questions to get an answer for a proper design:
1. What is the default value according to the ZigBee `ZLLLightLevel` sensor specification?
2. Which value do we need elsewhere (e.g. in the properties `tholddark` and `tholdoffset`)?
3. What is the most convenient value the people will understand / work with?

Reply:
To 3: I think Lux is the better value to display instead of the hue-specific format because that is a defined SI-unit. Therefore you can use it in formulas if you need to for your automation.

Another option would be using a unit like Candela or Lumen. But these are not optimal for our problem because they must be calculated using some unknown variables, like, for example, distance.

Reply:
1. The [documentation of the ZigBee Cluster library on page 423f](https://www.nxp.com/docs/en/user-guide/JN-UG-3077.pdf) gives the illuminance in logarithmic form, same as the hue bridge.

2. the thresholds are based on the same logartithmic values.

I changed the illuminance to lux in the last commit, but i think having two channels providing both values would be best